src/grammar.coffee

```diff
@@ -533,6 +534,34 @@ grammar =
     o '...',                                     -> new Expansion
   ]
 
+  # The **ArgElisionList** is the list of objects, contents of an array literal
+  # (i.e. comma-separated expressions and elisions). Newlines work as well.
+  ArgElisionList: [
+    o 'ArgElision'
+    o 'ArgElisionList , ArgElision',                                      -> $1.concat $3
+    o 'ArgElisionList TERMINATOR Arg',                                    -> $1.concat $3
+    o 'ArgElisionList , TERMINATOR ArgElision',                           -> $1.concat $4
+    o 'INDENT ArgElisionList OptComma OUTDENT',                           -> $2
+    o 'INDENT ArgElisionList , Elisions OUTDENT',                         -> $2.concat $4
+    o 'ArgElisionList OptComma INDENT ArgElisionList OptComma OUTDENT',   -> $1.concat $4
+    o 'ArgElisionList , Elisions INDENT ArgElisionList OptComma OUTDENT', -> $1.concat $3, $5
+    o 'ArgElisionList OptComma INDENT ArgElisionList , Elisions OUTDENT', -> $1.concat $4, $6
+  ]
+
+  ArgElision: [
+    o 'Arg',                  -> [$1]
+    o 'Elisions Arg',         -> [].concat $1, $2
+  ]
+
+  Elisions: [
+    o 'Elision'
+    o 'Elisions Elision',     -> [].concat $1, $2
+  ]
+
+  Elision: [
+    o ',',                    -> new Elision
```

Comment:
How does this play nice with `OptComma`? Is that why I think some of the tests are wrong, because they're discarding the last comma as optional?

Reply:
Elision is `,` at the beginning of the array, e.g. `[,a]` (length is 2) or `,` preceded by `,`, e.g. `[a,,b]` (length is 3).
So, `OptComma` should not precede `Elision` in the `grammar`.